packages/services/src/services/Locale/Locale.js

```diff
@@ -152,8 +154,46 @@ class LocaleAPI {
    *    const locale = await LocaleAPI.getLang();
    * }
    */
-  static getLang() {
-    if (root.document.documentElement.lang) {
+  static async getLang() {
+    let ddoLocal = await DDOAPI.getAll();
+
+    if (ddoLocal && ddoLocal.page && ddoLocal.page.pageInfo) {
+      const lang = root.document.documentElement.lang || undefined;
```

Comment:
@reesedownes we might want to separate out the logic here a bit. Perhaps a private method for `getFromDDO`, and `getFromLang`, then `getLang` would just be doing one or the other, or ultimately `_localeDefault` if both fail.